SPIFlashFileSystem.device.lib.nut

```diff
@@ -933,7 +918,7 @@ class SPIFlashFileSystem.FAT {
     // Removes a file from the FAT
     function removeFile(fname) {
```

Comment:
It occurs to me that perhaps we should be more consistent across the classes on file specification: some methods use filename, others fileID, still other fileRef (ie. name or ID). We should probably settle on the latter so as not to break any existing code. I can run through and update the code, if you like.

Reply:
Shouldn't break code: they pass in a string or an int as before, and we just handle it under the hood. But yes, if this update needs to get out quickly, this is not the time for updates like that.